config/opal_setup_sphinx.m4

```diff
@@ -0,0 +1,61 @@
+dnl -*- shell-script -*-
+dnl
+dnl Copyright (c) 2020-2022 Cisco Systems, Inc.  All rights reserved.
+dnl
+dnl $COPYRIGHT$
+dnl
+dnl Additional copyrights may follow
+dnl
+dnl $HEADER$
+dnl
+
+AC_DEFUN([OPAL_SETUP_SPHINX],[
+    OPAL_VAR_SCOPE_PUSH([summary_msg])
+
+    # Quick check to see if we have already-built docs (e.g., if we're
+    # in a tarball vs. a fresh git clone).
+    AC_MSG_CHECKING([if pre-built docs are available])
+    AS_IF([test -f "$srcdir/docs/_build/man/MPI_T.5"],
+          [opal_install_docs=1
+           AC_MSG_RESULT([yes])],
+          [opal_install_docs=0
+           AC_MSG_RESULT([no])])
+
+    # To generate HTML docs + man pages, we need Sphinx.  If we have
+    # Sphinx, then we're able to install the docs (potentially
+    # overriding opal_install_docs from what it was set above).
+    AC_PATH_PROG([SPHINX_BUILD], [sphinx-build])
+    AS_IF([test -n "$SPHINX_BUILD" && test -z "`echo $SPHINX_BUILD | $GREP missing`"],
+          [ # If we're building, we're also installing, regardless of
+            # whether we found pre-build docs or not (above).
+           opal_install_docs=1],
+          [SPHINX_BUILD=
+           OPAL_MAKEDIST_DISABLE="$OPAL_MAKEDIST_DISABLE Sphinx/Documentation"
+           AC_MSG_WARN([*** Could not find a suitable sphinx-build on your system.])
```

Comment:
I don't think we should warn here.  Most people running configure will never run "make dist", but will see this warning.  I'd be fine with an `AC_MSG_NOTICE`, since NOTICE prints to stdout and is silenced by --quiet, whereas WARN goes to stderr and is not quieted.